schema/descriptor.schema.json

```diff
@@ -1,501 +0,0 @@
-{
```

Comment:
I prefer that as well, but it does seem to break the python package - it can only contain (to my knowledge) files that are downstream of the main directory containing `setup.py`. One option would be removing the validator in Ruby, as you said, and having the Python directory sit at the base of the repo. Then we could have the schema separate from the package, but still include it as it'd be downstream. Thoughts?